concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -171,39 +205,35 @@ def main():
         'receive',
         REQUESTOR_PRIVATE_KEY,
         CONCENT_PUBLIC_KEY,
-        headers = {
+        headers={
             'Content-Type': 'application/octet-stream',
             'concent-client-public-key': b64encode(REQUESTOR_PUBLIC_KEY).decode('ascii'),
-        }
+        },
+        expected_status=200
     )
 
+
     #  Test CASE 2B - Send ForceSubtaskResults with not enough amount of funds on account
     api_request(
         cluster_url,
         'send',
         PROVIDER_PRIVATE_KEY,
         CONCENT_PUBLIC_KEY,
         force_subtask_results(
-            timestamp = timestamp_to_isoformat(current_time),  # current_time
+            timestamp = current_timestamp,  # current_time
             ack_report_computed_task = ack_report_computed_task(
-                timestamp = timestamp_to_isoformat(current_time - 14401),  # current_time - 4:00:01
+                timestamp = ack_report_computed_task_timestamp,  # current_time - 4:00:10
```

Comment:
I see you changed the time in the comment. Was the comment wrong or is it an accidental change?

Reply:
I have changed the time, because I had introduced TIME_GUARD variable - a safety offset